bindings/python/tests/test_block.py

```diff
@@ -6,32 +6,231 @@
+                                                                                                                                                                                                "type": 2, "data": "0x0d25"}]}, {
+                                                                                                                                                                                                    "type": 3, "amount": "234100", "unlockConditions": [
+                                                                                                                                                                                                        {
+                                                                                                                                                                                                            "type": 0, "address": {
+                                                                                                                                                                                                                "type": 0, "pubKeyHash": "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"}}, {
+                                                                                                                                                                                                                    "type": 1, "returnAddress": {
+                                                                                                                                                                                                                        "type": 0, "pubKeyHash": "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"}, "amount": "234000"}]}, {
+                                                                                                                                                                                                                            "type": 3, "amount": "1000000", "unlockConditions": [
+                                                                                                                                                                                                                                {
+                                                                                                                                                                                                                                    "type": 0, "address": {
+                                                                                                                                                                                                                                        "type": 0, "pubKeyHash": "0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"}}, {
```

Comment:
Can we check if formatting is somehow configurable? This is nonsense

Reply:
ill do that in the issue when i address it fully